lib/helper/WebDriver.js

```diff
@@ -2186,58 +2193,57 @@ async function proceedSee(assertType, text, context, strict = false) {
   return stringIncludes(description)[assertType](text, selected);
 }
 
-// Mimic Array.forEach() API, but with an async callback function.
-// Execute each callback on each array item serially. Useful when using WebDriver API.
-//
-// Added due because of problem with chrome driver when too many requests
-// are made simultaneously. https://bugs.chromium.org/p/chromedriver/issues/detail?id=2152#c9
-//
-// @param {object[]} array Input array items to iterate over
-// @param {function} callback Async function to excute on each array item
-// @return Array
-async function forEachAsync(array, callback, option = {}) {
+/**
+ * Mimic Array.forEach() API, but with an async callback function.
+ * Execute each callback on each array item serially. Useful when using WebDriver API.
+ *
+ * Added due because of problem with chrome driver when too many requests
+ * are made simultaneously. https://bugs.chromium.org/p/chromedriver/issues/detail?id=2152#c9
+ * @private
```

Comment:
Made JSDoc format.

But it leads to creation definitions for this internal functions.
Added such code ignore by `@private` and `@ignore` tags in #2113